test/components/tree-view.spec.tsx

```diff
@@ -98,369 +70,387 @@ describe('<TreeView />', () => {
 
-        expandItemWithLabel(select, treeData[0].label);
-        nodeChildren!.forEach(child => expandItemWithLabel(select, child.label));
+        treeView.getItem(treeData[0].label).clickIcon();
+        nodeChildren!.forEach(child =>  treeView.getItem(child.label).clickIcon());
 
         await waitForDom(() => allNodesLabels.forEach(item =>
-            expect(select(treeView + '_DEMO', getTreeItem(item)), `item did not appear: ${item}`).to.be.present()));
+            expect(treeView.getItem(item).root, `item did not appear: ${item}`).to.be.present()));
 
-        const elementToSelect = select(treeView + '_DEMO', getTreeItem(allNodesLabels[2]));
-
-        selectItemWithLabel(select, allNodesLabels[2]);
-        return waitForDom(() => expect(isElementSelected(elementToSelect!, treeItemStyle)).to.equal(true));
+        treeView.getItem(allNodesLabels[2]).clickLabel();
+        await waitForDom(() => expect(treeView.getItem(allNodesLabels[2]).isSelected()).to.equal(true));
     });
 
     it('renders a tree view with custom children', async () => {
-        const {select, waitForDom} = clientRenderer.render(<TreeViewDemoCustom />);
+        const {driver: treeViewDemoCustom, waitForDom} = clientRenderer.render(
+            <TreeViewDemoCustom />
+        ).withDriver(TreeViewDemoCustomDriver);
+
+        const {customTreeView} = treeViewDemoCustom;
 
-        await waitForDom(() => expect(select(treeView + '_DEMO_CUSTOM'), 'custom demo not present').to.be.present());
+        await waitForDom(() => expect(customTreeView.root, 'custom demo not present').to.be.present());
 
         const nodeChildren = treeData[0].children;
-        await waitForDom(() => expect(select(getTreeItem(nodeChildren![1].label))).to.be.absent());
+        await waitForDom(() => expect(customTreeView.getItem(nodeChildren![1].label).root).to.be.absent());
 
-        expandItemWithLabel(select, treeData[0].label);
-        nodeChildren!.forEach(child => expandItemWithLabel(select, child.label));
+        customTreeView.getItem(treeData[0].label).clickIcon();
+        nodeChildren!.forEach(child => customTreeView.getItem(child.label).clickIcon());
 
         await waitForDom(() => allNodesLabels.forEach(item =>
-            expect(select(treeView + '_DEMO_CUSTOM', getTreeItem(item)),
+            expect(customTreeView.getItem(item).root,
                 `item did not appear: ${item}`).to.be.present()));
 
-        const elementToSelect = select(treeView + '_DEMO_CUSTOM', getTreeItem(allNodesLabels[2]));
-
-        selectItemWithLabel(select, allNodesLabels[2]);
-        return waitForDom(() => expect(isElementSelected(elementToSelect!, treeViewDemoStyle)).to.equal(true));
+        customTreeView.getItem(allNodesLabels[2]).clickLabel();
+        await waitForDom(() => expect(customTreeView
+                                        .getItem(allNodesLabels[2])
+                                        .isSelected(treeViewDemoStyle)).to.equal(true));
```

Comment:
I think the more logical way to go is to have a static `isItemSelected` method on `TreeViewDemoCustomDriver'